src/plugin/isBetween/index.js

```diff
@@ -0,0 +1,11 @@
+export default (o, c) => {
+  const proto = c.prototype
+  proto.isBetween = function (a, b) {
+    if (!(a instanceof c && b instanceof c)) {
+      throw new Error('isBetween arguments must be dayjs-like!')
+    }
+
+    return this.isAfter(a) && this.isBefore(b)
+  }
```

Comment:
also do we need `this.isAfter(b) && this.isBefore(a)` ?

Reply:
What you mean? It's just re-using exist methods. Tell me more 🙂

Reply:
Just not sure if we should check the two compared date in different order.

```
// the first one could be true, if we think it's useful
moment('2018-05-01').isBetween('2019-05-01', '2017-05-01') // false 
moment('2018-05-01').isBetween('2017-05-01', '2019-05-01') // true
```

Reply:
Oh, I see. Give me few time.

Reply:
see your latest commit, cool.

still, it's just a discussion of my idea not a decision

Reply:
Why? If we working just with two dates it is really simple to compare our date with they. Or you mean more than 2 arguments?

Reply:
no, just a discussion about if we should ignore the given order or not.